src/commands/jobs/add.ts

```diff
@@ -52,21 +50,25 @@ export = {
 	},
 	handler: argv => {
 		runCommand(argv, async (client, output) => {
-			let job: any = {
-				kind: argv.kind,
-				name: argv.name,
-				schedule: {
-					mode: argv.manual ? 'manual' : 'auto'
-				}
+			let jobCliParameters: any = {
+				...argv.kind && { kind: argv.kind },
+				...argv.name && { name: argv.name },
+				...argv.manual && { schedule: { mode: 'manual' }}
 			};
-
-			let options = argv.optionsInput ? await readJsonInput() : undefined;
-			if (!options && argv.optionsFile) {
-				options = JSON.parse(fs.readFileSync(argv.optionsFile, 'utf-8'));
+			
+			let jobParametersInput = argv.jobInput ? await readJsonInput() : undefined;
+			if (!jobParametersInput && argv.jobFile) {
+				jobParametersInput = JSON.parse(fs.readFileSync(argv.jobFile, 'utf-8'));
 			}
-			job[job.kind] = options;
 
-			if (!options && !argv.optionsInput && job.kind === 'transfer' && argv.wizard) {
+			let jobDefaults: any = {
+				kind: 'transfer',
+				schedule: { mode: 'auto' }
```

Comment:
I would suggest defaulting the mode to manual

Reply:
the default mode was already auto.  i didn't change that as part of this code change.  is there a reason we should change from our standard?

Reply:
from what I recall, the jobs created via CLI where all manual (so maybe that original default was not working). Most use cases for creating jobs via scripting will be to create jobs ahead of time, but not executing them.  If this was set to 'auto' as a default, I would most likely override it and set to manual for the standard use scripts.